snuba/query/snql/parser.py

```diff
@@ -319,8 +338,13 @@ def visit_match_clause(
         ):
             join_clause = build_join_clause(match)
             return join_clause
+        if not isinstance(match, (QueryEntity, QueryStorage)):
+            import pdb
+
+            pdb.set_trace()
+            print(match)
 
-        assert isinstance(match, QueryEntity)  # mypy
+        assert isinstance(match, (QueryEntity, QueryStorage))  # mypy
```

Comment:
random comment that you can ignore: if you care to read a relevant excerpt of the pragmatic programmer about "assertive programming" https://www.informit.com/articles/article.aspx?p=2982114&seqNum=3